pyguide.md

```diff
@@ -2009,9 +2009,11 @@ import tensorflow as tf
 from book.genres import scifi
 from myproject.backend.hgwells import time_machine
 from myproject.backend.state_machine import main_loop
-from otherproject.ai import body
-from otherproject.ai import mind
-from otherproject.ai import soul
+from otherproject.ai import (
```

Comment:
It looks like this fits on a 80 character line, agree with this rule once the line is 80+

```suggestion
from otherproject.ai import body, mind, soul
```

Reply:
I'm kinda torn between here. On one hand multiple imports per line are more compact and look better. On the other, it's easier to track modifications in code with one import per line.

Reply:
now i'm torn too :P

One line one thought > compact code

so maybe leave as igor proposed?

Reply:
I'm fine either way